Kanban/Controllers/KBNProjectDetailViewController.m

```diff
@@ -12,23 +12,41 @@
 
 #define TABLEVIEW_TASK_CELL @"TaskCell"
 #define SEGUE_TASK_DETAIL @"taskDetail"
+#define SEGUE_ADD_TASK @"addTask"
 
 
 @interface KBNProjectDetailViewController ()
 
 @property (weak, nonatomic) IBOutlet UITableView *tableView;
 
+@property (strong, nonatomic) IBOutlet UILongPressGestureRecognizer *longPress;
+@property (strong, nonatomic) IBOutlet UITapGestureRecognizer *doubleTap;
+@property (strong, nonatomic) IBOutlet UITapGestureRecognizer *tap;
+
+@property (assign, nonatomic) BOOL cellSelected;
+
 @end
 
-@implementation KBNProjectDetailViewController
+@implementation KBNProjectDetailViewController {
+    CGPoint beginPoint, endPoint;
```

Comment:
Definí esas variables ahi porque son auxiliares a un solo método, que las
necesito para determinar si hubo un swipe después del long press. Guardan
el punto cuando puse el dedo y cuando lo levanté. Me pareció que no
calificaban conceptualmente como "properties" del view controller.

On Wed, Apr 29, 2015 at 9:22 AM, nporpigliaglobant <notifications@github.com
> wrote:

> In Kanban/Controllers/KBNProjectDetailViewController.m
> <https://github.com/nporpigliaglobant/Kanban/pull/14#discussion_r29330140>
> :
>
> >  @end
> >
> > -@implementation KBNProjectDetailViewController
> > +@implementation KBNProjectDetailViewController {
> > +    CGPoint beginPoint, endPoint;
>
> Por qué algunas variables estan definidas con @property
> <https://github.com/property> y otras acá? Definir todas como properties,
> aunque sean privadas
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nporpigliaglobant/Kanban/pull/14/files#r29330140>.
>

-- 


The information contained in this e-mail may be confidential. It has been 
sent for the sole use of the intended recipient(s). If the reader of this 
message is not an intended recipient, you are hereby notified that any 
unauthorized review, use, disclosure, dissemination, distribution or 
copying of this communication, or any of its contents, 
is strictly prohibited. If you have received it by mistake please let us 
know by e-mail immediately and delete it from your system. Many thanks.

 

La información contenida en este mensaje puede ser confidencial. Ha sido 
enviada para el uso exclusivo del destinatario(s) previsto. Si el lector de 
este mensaje no fuera el destinatario previsto, por el presente queda Ud. 
notificado que cualquier lectura, uso, publicación, diseminación, 
distribución o copiado de esta comunicación o su contenido está 
estrictamente prohibido. En caso de que Ud. hubiera recibido este mensaje 
por error le agradeceremos notificarnos por e-mail inmediatamente y 
eliminarlo de su sistema. Muchas gracias.